src/ledger/database/interface.zig

```diff
@@ -546,5 +547,169 @@ fn tests(comptime Impl: fn ([]const ColumnFamily) type) type {
 
             try std.testing.expectEqual(null, try iter.next());
         }
+
+        pub fn @"iterator forward start before all"() !void {
```

Comment:
We should merge #353 and then rebase + migrate these tests.

Reply:
e2910ce044b991a4efd475b5c95bae141ffa5d67